pkg/infoschema/infoschema_v2.go

```diff
@@ -444,6 +440,17 @@ func isSpecialDB(dbName string) bool {
 		dbName == util.MetricSchemaName.L
 }
 
+// EvictTable is exported for testing only.
+func (is *infoschemaV2) EvictTable(schema, tbl string) {
```

Comment:
maybe implement it in test file of the same package, see https://github.com/pingcap/tidb/blob/ef0a3eb8af93da86fcf56443e58f0363584443e0/pkg/disttask/framework/scheduler/main_test.go#L25-L32

Reply:
Some other parts may also need this functionality and it's in some other packages.
https://github.com/pingcap/tidb/pull/54479/files#diff-cabf59f012600c8b513d4eb03385f48e3fdd9b8ac3f13c175133d40c9354722bR279